### DISCUSSION TITLE
Single letter keyboard shortcut like Focus F doesn't work when focusing Scene tree, selects node starting with that letter in scene tree (or do nothing) instead

### DISCUSSION THREAD

**Discussion by hsandt** [godotengine/godot-proposals#6853] [Editor]
Most of the time, when I use the Focus shortcut F I have selected the node in the Scene tree (otherwise I would already have it in scene view). But pressing a single-letter keyboard shortcut while focusing on the Scene tree panel will find the next node whose name starts with that letter (e.g. Floor) instead. Sometimes, it would do nothing. I tried with other letters and it just did nothing anyway, for some reason.

If you click on the Scene view first then you can use F (and probably other single-letter shortcuts) reliably, but it's cumbersome.

Mapping to another shortcut like F4 works reliably.

Should we give priority to the shortcut if it exists, over node name search then?
Or just set the default to something else like F4, or Ctrl+Alt+F (more complex...) ?

I think other editors just give priority to shortcuts when meaningful.

**Comment by Zireael07** [comment]
I think it's a bug not a proposal

**Comment by adamscott** [comment]
I agree with @Zireael07. Could you create [an issue in the godot repository](https://github.com/godotengine/godot/issues) instead?

**Reply by hsandt** [reply]
Sure, here it is: https://github.com/godotengine/godot/issues/76947

I noticed that even F4 doesn't wore reliably now so I changed the issue description too.
